app/assets/javascripts/controllers/ops/pglogical_replication_form_controller.js

```diff
@@ -277,7 +277,71 @@ ManageIQ.angular.app.controller('pglogicalReplicationFormController', ['$http',
       return true;
     else
       return false;
-  }
+  };
+
+  var $ctrl = this;
+
+  $ctrl.animationsEnabled = true;
+  $ctrl.ssh_params = {ssh_host: "", ssh_user: "", ssh_password: ""};
+
+  $scope.isCentralAdminEnabled = function(idx) {
+    return $scope.pglogicalReplicationModel.subscriptions[idx].auth_key_configured;
+  };
+
+  $scope.enableCentralAdmin = function(idx) {
+    var data = {};
+    data["provider_region"] = $scope.pglogicalReplicationModel.subscriptions[idx].provider_region;
+    data["ssh_host"] = $ctrl.ssh_params.ssh_host;
+    data["ssh_user"] = $ctrl.ssh_params.ssh_user;
+    data["ssh_password"] = $ctrl.ssh_params.ssh_password;
+
+    miqService.sparkleOn();
+    var url = "/ops/enable_central_admin";
+    miqService.miqAjaxButton(url, data);
+  };
+
+  $scope.disableCentralAdmin = function(idx) {
+    if (confirm("Are you sure you want to Disable Central Admin for this Region?")){
+      miqService.sparkleOn();
+      var url = "/ops/disable_central_admin/";
+      var data = {};
+      data["provider_region"] = $scope.pglogicalReplicationModel.subscriptions[idx].provider_region;
+      miqService.miqAjaxButton(url, data);
+    }
+  };
+
+  $scope.launchAuthKeyModal = function (idx) {
+    $ctrl.ssh_params.ssh_host = $scope.pglogicalReplicationModel.subscriptions[idx].remote_ws_address;
```

Comment:
Is this pulling the address set [here](https://github.com/ManageIQ/manageiq/pull/11646/files#diff-84dbfcd1a7ac9f7442c66dadf41485bbR186)?